package.json

```diff
@@ -54,12 +54,12 @@
 		"@namics/stylelint-bem": "^8.0.0",
 		"@rushstack/eslint-patch": "^1.2.0",
 		"@types/jest": "^29.2.3",
-		"@types/lodash": "^4.14.187",
+		"@types/lodash": "^4.14.191",
 		"@typescript-eslint/eslint-plugin": "^5.45.0",
 		"@typescript-eslint/parser": "^5.45.0",
 		"@vitejs/plugin-vue": "^2.3.4",
 		"@vue/eslint-config-typescript": "^11.0.2",
-		"@vue/test-utils": "v2.2.1",
+		"@vue/test-utils": "^2.2.5",
```

Comment:
Oh neat, a mirrored caret operator!

Reply:
it's fascinating that this still worked. I guess it was then effectively pinned to that particular version?

Reply:
Looks like it, yeah – on https://semver.npmjs.com/ a leading “v” seems to be accepted and ignored.